kotori.js.org

BIYUEHU opened the pull request:
- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```kotori.js.org``` to make it work. 
FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages

I'll merge after we have "contact". Please keep this PR open.

BIYUEHU commented:
> I've already added your requested subdomain to JS.ORGs zonefile. But you have to (re-)add a file named ```CNAME``` to your branch with the single line ```kotori.js.org``` to make it work. 
> FYI: https://help.github.com/articles/configuring-a-publishing-source-for-github-pages
> 
> I'll merge after we have "contact". Please keep this PR open.

Thanks.I have already updated file ```CNAME``` now.

indus commented:
Welcome https://kotori.js.org
